### PR TITLE
Testing for occur_prob.py

### PR DESCRIPTION
**Pull request by jklenzing**
Addresses #51.

Summary of changes:
- Unit tests added for `ssnl.occur_prob`
- Bugs in 3D routines fixed
- PEP8 / style

### CODE REVIEW

File: pysat/tests/test_ssnl_occur_prob.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+class TestBasics():
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+        orbit_info = {'index': 'longitude', 'kind': 'longitude'}
+        self.testInst = pysat.Instrument('pysat', 'testing',
+                                         clean_level='clean',
+                                         orbit_info=orbit_info)
+        self.testInst.bounds = (pysat.datetime(2008, 1, 1),
+                                pysat.datetime(2008, 3, 31))
```
Comment (aburrell) [inline pysat/tests/test_ssnl_occur_prob.py]: Why not test for a month instead of 3 months?  Will probably speed things up.

Comment (jklenzing) [reply pysat/tests/test_ssnl_occur_prob.py]: I'll give it a try.  Takes about 41 seconds now.

Comment (jklenzing) [reply pysat/tests/test_ssnl_occur_prob.py]: Need to increase the tolerance for one test from 1.0e-2 to 2.0e-2, but cuts the tests for this file down to 15 seconds.  Happy to go either way on this.

**Comment by aburrell** [comment]
Our tests are very long, so I vote for shortening the time.

**Comment by jklenzing** [comment]
No problem.  Turning my attention to the unit tests again for the near future.  Stay tuned...

**Pull request merged**